pkg/aks/components/AksNodePool.vue

```diff
@@ -387,28 +387,26 @@ export default defineComponent({
           v-if="(taints && taints.length) || isView"
           class="taints"
         >
-          <thead>
-            <tr>
-              <th>
-                <label class="text-label">
-                  {{ t('aks.nodePools.taints.key') }}
-                  <span class="text-error">*</span>
-                </label>
-              </th>
-              <th>
-                <label class="text-label">
-                  {{ t('aks.nodePools.taints.value') }}
-                  <span class="text-error">*</span>
-                </label>
-              </th>
-              <th>
-                <label class="text-label">
-                  {{ t('aks.nodePools.taints.effect') }}
-                </label>
-              </th>
-              <th />
-            </tr>
-          </thead>
+          <tr>
```

Comment:
it looks like this and a few other similar changes below are reverts of recent changes